ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/viewcreation/valueobjects/LdesConfig.java

```diff
@@ -98,4 +106,13 @@ public Model getDcat() {
 		return dcat;
 	}
 
+	public void setDefaultView(boolean defaultView) {
+		this.defaultView = defaultView;
+		this.viewConfig.setDefaultView(defaultView);
+	}
```

Comment:
Could this be refactored to only keep defaultView in either `LdesConfig` or `ViewConfig`?